### PR TITLE
Fix localization and added Deutsch(German) local

### PR DESCRIPTION
**Pull request by amanalili**
Must also include PR from react-gui: https://github.com/EdgeApp/edge-react-gui/pull/2448

### CODE REVIEW

File: packages/edge-login-ui-rn/src/common/locales/strings.js [hunk 1]
```diff
@@ -39,9 +45,6 @@ export function selectLocale(locale: string = 'en'): boolean {
   let found = false
   const lang = normalizedLocale.slice(0, 2)
 
-  // Set default of US English
```
Comment (paullinator) [inline packages/edge-login-ui-rn/src/common/locales/strings.js:42]: Why was this removed?

**Review by paullinator — CHANGES_REQUESTED** [review]

File: packages/edge-login-ui-rn/src/common/locales/strings.js (hunk 1 above)
Comment (amanalili) [reply packages/edge-login-ui-rn/src/common/locales/strings.js:42]: This is copy pasted from the react-gui strings.js to login-ui strings.js. So any changes are just replicated

**Review by paullinator — APPROVED** [review]
